frontend/VolunteerOne/screens/Friends/notificationsPage.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { StyleSheet, Dimensions, ScrollView } from 'react-native';
+import { Block, theme } from 'galio-framework';
+import { Button } from "../../components";
+
+// import { Card } from '../../components';
+import NotificationItem from '../../components/NotificationItem';
+// import articles from '../../constants/articles';
+import notifications from '../../constants/notifications';
+
+// <matt>
+// import { Block, Button, Text, theme } from "galio-framework";
+// </matt>
+
+const { width } = Dimensions.get('screen');
+
+class Home extends React.Component {
+  renderNotifications = () => {
+    return (
+      <ScrollView
+        showsVerticalScrollIndicator={false}
+        contentContainerStyle={styles.notifications}>
+ 
+        <Block flex>
+          <NotificationItem item={notifications[0]} horizontal  />
```

Comment:
As the notifications are in an array, could they potentially be looped through instead? You could consider using the ```map``` function to improve readability/testing with other data later on

Reply:
oops, i deleted this notifications page, the looped version should have been added in the previous PR